pkg/controller/statefulset/stateful_pod_control.go

```diff
@@ -314,11 +314,11 @@ func (spc *StatefulPodControl) recordClaimEvent(verb string, set *apps.StatefulS
 	}
 }
 
-// createPersistentVolumeClaims creates all of the required PersistentVolumeClaims for pod, which must be a member of
```

Comment:
Don't make this public. Encapsulate the volume re-creation in the implementation of this interface.